.cirrus.yml

```diff
@@ -315,6 +316,7 @@ special_testing_rootless_task:
     timeout_in: 60m
 
     setup_environment_script: '$SCRIPT_BASE/setup_environment.sh |& ${TIMESTAMP}'
+    system_test_script: '$SCRIPT_BASE/system_test.sh |& ${TIMESTAMP}'
     integration_test_script: '$SCRIPT_BASE/integration_test.sh |& ${TIMESTAMP}'
```

Comment:
(Here and elsewhere) It's hurting my brain that we're running system tests before integration tests.

Is it reasonable to think, a new system-test, could perturb the environment such that it passes, but leads to a future integration-test failure for a non-obvious reason?

In other words, the int. tests are already difficult to debug, if we push folks toward writing new system-tests, could that be shooting our integration-tests in the foot?

---even if not---

It might be worth reversing the order temporarily to see if the oddball integration-test flakes go away :smile:

Reply:
I did that because I thought a 3-minute test would be nice to run before a 30-minute one; quicker notice of failure and all that. But I'll change it and see.

Reply:
Ya I suspected as much.  The last run failed in the same way, same test.  Let's see if it does that a third time now with the order reversed...